src/etcd/kv.cr

```diff
@@ -126,6 +126,40 @@ module Etcd
       Model::TxnResponse.from_json(response.body).succeeded
     end
 
+    def move(key : String,  key_d : String, value, lease : Int64 = 0_i64) : Bool
```

Comment:
As this is a constructed transaction, could you describe the operation?

Reply:
sometime, we may have different class to store data, something like queue,

Reply:
sometime we may store data with different key prefix, some thing like different queues.

Reply:
Sometime we may store data with different key prefix, some thing like different queues. When we handle the data, we have demand to add  a new key with the same value, and at the same time, we need to delete the old key-value. We need to do them at the same time. so add function move to resolve it.
thanks.